cfme/tests/storage/test_volume_snapshot.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from selenium.common.exceptions import TimeoutException
```

Comment:
**Required**

Remove this import, you actually want `cfme.utils.wait.TimedOutError`